PYME/Analysis/Tracking/trackUtils.py

```diff
@@ -41,10 +41,13 @@ def __getitem__(self, key):
             return ''
             
         data = self.clump[key]
+
+        if len(data.shape) > 2:
```

Comment:
Is this the right behaviour / check to be performing? Should it be `>=2`? Should we warn? 
Should we (eventually) plot a heatmap/kymograph for dim=2?

Reply:
I'm not 100% sure how this is supposed to work, but it does work, looking like the graphs in the PDF, with `>2`. However, no variables with `dim==2` were shown in the example images. `>=2` or `>1` would also be fine. kymograph for `dim==2` seems like the right move, based on what I've understood.